libs/pageserver_api/src/models.rs

```diff
@@ -23,13 +23,28 @@ pub enum TenantState {
     Active {
         background_jobs_running: bool,
     },
-    /// A tenant is recognized by pageserver, but not yet ready to operate:
-    /// e.g. not present locally and being downloaded or being read into memory from the file system.
+    /// A tenant is recognized by pageserver, but it is being detached or the system is being
+    /// shut down.
     Paused,
-    /// A tenant is recognized by the pageserver, but no longer used for any operations, as failed to get activated.
+    /// A tenant is recognized by the pageserver, but can no longer used for any operations,
+    /// because it failed to get activated.
     Broken,
 }
 
+impl TenantState {
+    pub fn has_in_progress_downloads(&self) -> bool {
```

Comment:
Nice. Had a similar change in https://github.com/neondatabase/neon/pull/2872 to fix the flaky tests, but this function is better than what I did.